### PR TITLE
flex

### PR DESCRIPTION
**Pull request by zarva6596**
[DEMO LINK](https://zarva6596.github.io/layout_snake/)
[TEST REPORT LINK](https://zarva6596.github.io/layout_snake/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
-  <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Snake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
```
Comment (irenhh) [inline src/index.html:6]:
```suggestion
    <meta 
      name="viewport"
      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
    >
```
formatting

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
+body {
+  margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 100px;
```
Comment (irenhh) [inline src/style.css:4]: use this value for those elements you need

**Review by irenhh — CHANGES_REQUESTED** [review]
check the page for big screens, you have white spaces on edges

no need to repeat orders if they don't change

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+}
+
+.flex {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 0 auto;
+}
+
+@media screen and (min-width: 600px) and (max-width: 900px) {
```
Comment (vsm2k17) [inline src/style.css:53]: no need to set min and max, one thing is enough, (min-width in your case)

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+.box5 {
+  background-color: rgb(51, 0, 0);
+  order: 5;
+}
+
+.box6 {
+  background-color: rgb(0, 0, 0);
+  order: 6;
+}
```
Comment (vsm2k17) [inline src/style.css:44]: no need to repeat orders if they don't change, 
in this case, they already in right order.

**Review by mykola-yarchuk — CHANGES_REQUESTED** [review]

**Review by danheim — APPROVED** [review]

**Pull request closed without merging**